src/Nethermind/Nethermind.Facade/Simulate/SimulateBridgeHelper.cs

```diff
@@ -106,10 +109,15 @@ private bool TrySimulate(
                 suggestedBlocks[0] = currentBlock;
 
                 IBlockProcessor processor = env.GetProcessor(payload.Validation, spec.IsEip4844Enabled ? blockCall.BlockOverrides?.BlobBaseFee : null);
-                Block processedBlock = processor.Process(stateProvider.StateRoot, suggestedBlocks, processingFlags, tracer)[0];
+                Block processedBlock = processor.Process(stateProvider.StateRoot, suggestedBlocks, processingFlags, blockTracer)[0];
 
                 FinalizeStateAndBlock(stateProvider, processedBlock, spec, currentBlock, blockTree);
-                CheckMisssingAndSetTracedDefaults(simulateOutputTracer, processedBlock);
+
+                // if (tracer is SimulateBlockTracer simulateBlockTracer)
+                // {
+                //     CheckMisssingAndSetTracedDefaults(simulateBlockTracer.Results, processedBlock);
+                // }
```

Comment:
This is still probably crucial for main simulate, maybe we can do it on block tracer level for it?